### ISSUE TITLE
Default gradle build stuck without feedback.

### ISSUE THREAD

**Issue by jgsuess** [wala/WALA#342]
I am performing an exploration of Wala for our reserch lab. We have tried to run pre-compiled binary components to [test Wala examples](https://github.com/wala/WALA/wiki/Getting-Started#running-wala-example-programs), but where faced with issues. So I have decided to build from source to get latest.

I am running the standard gradle build using ``gradlew assembly`` as described in [the build instructions](https://github.com/wala/WALA/blob/master/README-Gradle.md). I have the required prerequisites and have a proxy set up so packages can be pulled.

The build fails to start both with and without the daemon. I have enabled debug-level logging ``--debug``, but there is no feedback from the system. Are there any other conditions that need to be met as prerequisites?

**Comment by liblit** [comment]
What output is produced leading up to the “stuck without feedback” point? Is there any initial output at all, or is it completely silent from the start?

FYI, your `gradlew assembly` should be `./gradlew assemble`, though that shouldn’t cause things to get stuck.

**Comment by msridhar** [comment]
FWIW when I've run into this issue before with gradle, it was because some plugin was unable to pull some network artifact.  Also note that the initial build with gradle causes some huge amount of Eclipse-related stuff to get downloaded, with no progress indicator.  It's taken several minutes to complete for me before.

**Comment by msridhar** [comment]
And yes any partial output would be useful, for comparison purposes.

**Comment by jgsuess** [comment]
There is no output. The build prompts ``starting daemon``, and then there is no more feedback. Even if I set the switch ``--no-daemon``, I see the same result. Gradle produces the lock files in the ``.gradle/daemon/(version)`` directory, but no log.
I have traced this back to being a proxy issue. However JVM proxy via environment and system property did not solve it. But when I ran using direct internet using my mobile, the build started. So this is a ``gradle`` problem, not a ``wala`` problem.
